chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -97,7 +97,7 @@ if [ $ret != 0 ] ; then
     exit $ret
 fi
 
-log "Restarting services for Rocky"
```

Comment:
why did you remove this log line?

Reply:
Because it's no longer true (no Rocky when upgrading to SOC9) ... and anyway it does not have much sense, the script actions are logged already

Reply:
OK, I might be wrong about Rocky and SOC9. But I still think the line does not bring any value (esp. when services are not _always_ restarted, there are further checks, see below)